src/nerve_tools/configure_nerve.py

```diff
@@ -187,14 +186,34 @@ def parse_args(args):
     parser = argparse.ArgumentParser()
     parser.add_argument('-f', '--heartbeat-path', default="/var/run/nerve/heartbeat",
                         help='path to nerve heartbeat file to monitor')
-    parser.add_argument('-s', '--heartbeat-threshold', type=int, default=NERVE_REGISTRATION_DELAY_S * 2,
+    parser.add_argument('-s', '--heartbeat-threshold', type=int, default=60,
                         help='if heartbeat file is not updated within this many seconds then nerve is restarted')
+    parser.add_argument('--nerve-config-path', type=str, default='/etc/nerve/nerve.conf.json')
+    parser.add_argument('--nerve-backup-command', type=json.loads, default='["service", "nerve-backup"]')
+    parser.add_argument('--nerve-command', type=json.loads, default='["service", "nerve"]')
+    parser.add_argument('--nerve-registration-delay-s', type=int, default=30)
+    parser.add_argument('--zk-topology-dir', type=str, default='/nail/etc/zookeeper_discovery')
+    parser.add_argument('--zk-location-type', type=str, default='superregion',
+                        help="What location type do the zookeepers live at?")
+    parser.add_argument('--zk-cluster-type', type=str, default='infrastucture')
+    parser.add_argument('--hacheck-port', type=int, default=6666)
+    parser.add_argument('--weight', type=int, default=CPUS,
```

Comment:
I'm not going to address this for now, since currently we're hard-coding and I'd like to get this change out.